react/components/PickupPointHour.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { FormattedTime } from 'react-intl'
+
+interface Props {
+  time: string
+}
+
+const PickupPointHour: React.FC<Props> = (props) => {
+  const hoursNumbers = props.time.split(':').map((t) => parseInt(t, 10)) as [
+    number,
+    number,
+    number
+  ]
+
+  const time = new Date().setHours(...hoursNumbers)
+
+  return <FormattedTime value={time} />
+}
+
+PickupPointHour.propTypes = {
+  time: PropTypes.string.isRequired,
+}
```

Comment:
Do we still need to maintain proptypes?

Reply:
I don't think so, but as the majority of the components here are still in javascript it may be helpful.. but what do you think, should I remove it?

Reply:
I think we should remove it to avoid the duplication with the props interface. But I'm unaware of the benefits of keeping the propTypes on runtime (none?) or DX (IDE probably fallback to TS props?)

Reply:
we can also use the prop types to generate the `interface Props` to eliminate the duplication, the `prop-types` already has an util for that